This patch makes clear what is the actual behavior in RHEL due to bug #38402.

Author: Matus Honek <mhonek@redhat.com>
Resolves: #1498841

--- a/doc/man/man5/ldap.conf.5
+++ b/doc/man/man5/ldap.conf.5
@@ -25,7 +25,7 @@ in their home directory which will be used to override the system-wide
 defaults file.
 The file
 .I ldaprc
-in the current working directory is also used.
+in the current working directory is \fBNOT\fP used (this differs from upstream).
 .LP
 .LP
 Additional configuration files can be specified using
@@ -50,7 +50,7 @@ Thus the following files and variables are read, in order:
 .nf
     variable     $LDAPNOINIT, and if that is not set:
     system file  ETCDIR/ldap.conf,
-    user files   $HOME/ldaprc,  $HOME/.ldaprc,  ./ldaprc,
+    user files   $HOME/ldaprc,  $HOME/.ldaprc,  (\fBNOT\fP ./ldaprc, see above),
     system file  $LDAPCONF,
     user files   $HOME/$LDAPRC, $HOME/.$LDAPRC, ./$LDAPRC,
     variables    $LDAP<uppercase option name>.
